app/src/main/java/com/teamagam/gimelgimel/app/sensor/viewModel/SensorDetailsViewModel.java

```diff
@@ -2,17 +2,46 @@
 
 import com.teamagam.gimelgimel.app.common.base.ViewModels.BaseViewModel;
 import com.teamagam.gimelgimel.app.map.model.geometries.PointGeometryApp;
+import com.teamagam.gimelgimel.domain.map.entities.geometries.PointGeometry;
+import com.teamagam.gimelgimel.domain.messages.entity.contents.SensorMetadata;
+import com.teamagam.gimelgimel.domain.sensors.DisplaySensorsInteractor;
+import com.teamagam.gimelgimel.domain.sensors.DisplaySensorsInteractorFactory;
+
+import javax.inject.Inject;
 
 public class SensorDetailsViewModel extends BaseViewModel {
 
-    private static int sCount = 3;
+    @Inject
+    DisplaySensorsInteractorFactory mSensorsDisplayInteractorFactory;
 
     private String mSensorName;
     private PointGeometryApp mSensorLocation;
+    private boolean mIsAnySelected;
+    private DisplaySensorsInteractor mDisplaySensorsInteractor;
 
-    public SensorDetailsViewModel() {
+    @Inject
+    SensorDetailsViewModel() {
         mSensorLocation = PointGeometryApp.DEFAULT_POINT;
```

Comment:
For it to have a default value and not crash before actual data is displayed.
It's an alternative for null checking.
The data will not actually be shown since the detail panel is GONE until some data exists.